src/system/ThemeModule/Resources/views/Admin/var.html.twig

```diff
@@ -0,0 +1,18 @@
+{% form_theme form with [
+'ZikulaFormExtensionBundle:Form:bootstrap_3_zikula_admin_layout.html.twig',
+'ZikulaFormExtensionBundle:Form:form_div_layout.html.twig'
```

Comment:
Would it also be possible to allow a "helptext" parameter to display a little explanation below each setting? (Using Bootstrap's `help-block`?)